students/JasneetChandok/Lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,21 @@
+'''
+Lesson: 02
+Excercise: Fizz Buzz
+Student Name: Jasneet Chandok
+Last Update Date: 04/18/19
+'''
+#!/usr/bin/env python3.6

```

Comment:
if you have a line like this, it should go at the very top of the file, it will only do any good on the first line.